app/controllers/api/feedback_controller.rb

```diff
@@ -5,6 +5,20 @@ class FeedbackController < ApiController
     before_action :authorize_user
     load_and_authorize_resource :feedback
 
+    def index
+      project = Project.find_by(identifier: url_params[:identifier])
```

Comment:
We're loading up the Project here and then loading it again inside `feedback_params`, it might be better to memoize the Project with something like

```
def project
  @project ||= Project.find_by(identifier: url_params[:identifier])
end
```